applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -17,27 +16,30 @@ def __init__(self, Model, settings ):
                 "model_part_name":"PLEASE_CHOOSE_MODEL_PART_NAME",
                 "mesh_id": 0,
                 "inlet_phi": 1.0,
-                "velocity_infinity": [1.0,0.0,0]
+                "velocity_infinity": [1.0,0.0,0],
+                "initialize_flow_field": true
             }  """ );
 
 
         settings.ValidateAndAssignDefaults(default_parameters);
 
         self.model_part = Model[settings["model_part_name"].GetString()]
+        self.fluid_model_part = self.model_part.GetRootModelPart()
         self.velocity_infinity = KratosMultiphysics.Vector(3)#array('d', [1.0, 2.0, 3.14])#np.array([0,0,0])#np.zeros(3)#vector(3)
```

Comment:
```suggestion
        self.velocity_infinity = settings["velocity_infinity"].GetVector()
```

I think that this should work so you can avoid to seat one by one each one of the components.